skins/cat17/src/scripts/donationComments.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * @licence GNU GPL v2+
+ * @author Gabriel Birke <gabriel.birke@wikimedia.de>
+ */
+
+/**
+ * Load and render current donation comments
+ *
+ * @constructor
+ */
+var DonationComments = function ( commentContainer, paginationContainer ) {
+	this.commentContainer = commentContainer;
+	this.paginationContainer = paginationContainer;
+};
+
+$.extend( DonationComments.prototype, {
+	init: function () {
+		this.commentContainer.html( 'Spendenkommentare werden geladen ...' );
```

Comment:
Being that guy, but can we please get copy from content via twig? For this page I don't see us getting a second chance to do that "while we are at it", but only as a task dedicated to i18n. A lost opportunity in my eyes.